src/Form/FormAutosuggest.jsx

```diff
@@ -283,8 +286,6 @@ FormAutosuggest.propTypes = {
   ignoredArrowKeysNames: PropTypes.arrayOf(PropTypes.string),
   /** Specifies loading state. */
   isLoading: PropTypes.bool,
-  /** An ARIA role describing the form autosuggest. */
-  role: PropTypes.string,
```

Comment:
We discussed this during the Paragon Working Group. We made the decision to remove consumers' ability to change the role to something that would break WCAG.